src/main/java/teammates/ui/template/StudentListTeamData.java

```diff
@@ -13,14 +13,12 @@
     private List<StudentListStudentData> students;
 
     public StudentListTeamData(TeamDetailsBundle team, Map<String, String> emailPhotoUrlMapping, String googleId,
-                               String sessionToken) {
+                               String sessionToken, String previousPage) {
```

Comment:
Use 8-space indentation

Reply:
I have to point out that I have noticed inconsistency with the 8-space indentation rule in already written code. I must ask out of genuine curiosity, is there a reason for this that I'm missing?